deb_build.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+create_file()
```

Comment:
@sandisamp I don't understand the requirement of this PR. Why we are adding build script for Debian where we don't have any build script at all? We are already using alien to convert RPM to deb then why we are adding this script? Did you find any problem in alien converted DEB?